src/devices/Ssd1351/Command/Commands.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Ssd1351.Command
+{
+    public enum Ssd1351Command : byte
+    {
+        SetColumn = 0x15,
+        WriteRam = 0x5C,
+        SetRow = 0x75,
+        SetRemap = 0xA0,
+        SetStartLine = 0xA1,
+        SetDisplayOffset = 0xA2,
+        SetNormalDisplay = 0xA6,
+        SetInvertedDisplay = 0xA7,
+        SelectFunction = 0xAB,
+        SetDisplayOff = 0xAE,
+        SetDisplayOn = 0xAF,
+        SetPrecharge = 0xB1,
+        SetClockDiv = 0xB3,
+        SetVSL = 0xB4,
+        SetGPIO = 0xB5,
+        SetPrecharge2 = 0xB6,
+        SetDeselectVoltageLevel = 0xBE,
+        SetContrastABC = 0xC1,
+        SetContrastMasterCurrent = 0xC7,
+        SetMultiplexorRatio = 0xCA,
+        SetCommandLocks = 0xFD,
+
+        // the following values are unused within the binding but may be used by application software
+        ReadRam = 0x5D, // Note that this is not used with an SPI interface as the SSD1351 cannot be read
+        SetDisplayAllOff = 0xA4,
+        SetDisplayAllOn = 0xA5,
+        SetDisplayEnhancement = 0xB2,
+        SetGreyLevels = 0xB8,
+        SetDefaultGreyLevels = 0xB9,
+        SetPrechargeLevel = 0xBB,
+        SetHorizontalScroll = 0x96,
+        StopScroll = 0x9E,
+        StartScroll = 0x9F,
+    }
+
+    public interface ICommand
```

Comment:
There is already an issue about using ICommands [https://github.com/dotnet/iot/issues/463](https://github.com/dotnet/iot/issues/463) (In fact looks like you logged it :P)

I actually agree with you and @krwq in there of not using this as I don't like all of the extra boxing that this entails, plus other concerns like versioning the ICommand interface or having to new up a command every time you want to send one command. Perhaps all commands should just be structs(so that they don't have to allocate) and just return value tuples which are the bytes. What do you think?

Reply:
@joperezr 
I have replaced the ICommand methodology with method calls on the binding object for the latest PR update. This makes more sense to me right now